h2o-algos/src/main/java/hex/coxph/CoxPH.java

```diff
@@ -111,16 +111,25 @@ public CoxPH(boolean startup_once) {
       }
 
       if( _train != null ) {
-        int nonFeatureColCount = (_parms._start_column!=null?1:0) + (_parms._stop_column!=null?1:0); 
+        int nonFeatureColCount = (_parms._start_column!=null?1:0) + (_parms._stop_column!=null?1:0);
         if (_train.numCols() < (2 + nonFeatureColCount))
           error("_train", "Training data must have at least 2 features (incl. response).");
+        if (null != _parms._stratify_by) {
+          int stratifyColCount = _parms._stratify_by.length;
+          if (_train.numCols() < (2 + nonFeatureColCount + stratifyColCount))
+            error("_train", "Training data must have at least 1 feature that is not response and is not stratified."); 
```

Comment:
I understand this, but think this would be a bit clearer: "Training data must have at least 1 feature that is not a response and is not used for stratification."

Reply:
OK, let's make it a bit better